### ISSUE TITLE
Raise error when reading Solomon instance without specifying instance format

### ISSUE THREAD

**Issue by leonlan** [leonlan/VRPLIB#59]
I think it'll be common for users to use the `read_instance` function without explicitly specifying `instance_format='solomon'`.
What will happen is that the function returns an empty dictionary. There are no verifications (yet) in `parse_vrplib` #51, so it might be helpful to issue a warning instead.

**Comment by leonlan** [comment]
We could do something like `np.loadtxt`:

https://github.com/numpy/numpy/blob/8cec82012694571156e8d7696307c848a7603b4e/numpy/lib/npyio.py#L1061-L1068

**Comment by leonlan** [comment]
In #76 we managed to find a way to verify if an instance is in VRPLIB format or not. We can do the same for Solomon instances. The Solomon instances have a fixed file format:
```
<NAME>

VEHICLE
NUMBER	CAPACITY
<number> <capacity>

CUSTOMER
CUST. NO 	XCOORD.	YCOORD.	DEMAND	READY TIME	DUE DATE	SERVICE TIME
<data row 1>
...
```

- We ignore all empty lines. 
- Then we check if the first line contains "VEHICLE"
- We check that the second line contains "NUMBER" and "CAPACITY"
- We check that the third line contains two integer values
- We check that the fourth line contains "CUSTOMER"
- The fifth line contains the customer header names, but these are inconsistent in number of spaces. For example, sometimes `SERVICE TIME` is used and sometimes `SERVICE         TIME` is used. So I'm not sure how to check this.

**Comment by leonlan** [comment]
Closed by #83.
